README.md

```diff
@@ -62,6 +62,59 @@ go get -u github.com/lucassabreu/clockify-cli
 Go to the [releases page](https://github.com/lucassabreu/clockify-cli/releases) and download the pre-compiled
 binary that fits your system.
 
+Getting Started
+----
+
+After you install the CLI, run the `clockify config init` command to setup your environment variables. You’ll be prompted to enter your user information. You can get your clockify api token [here](https://clockify.me/user/settings).
```

Comment:
Not sure, but shouldn't the command be `clockify-cli config init`?

Reply:
this pr is a little out of date because of a change i made on the config command... to init the config you need to run `clockify-cli config --init` now...

Reply:
Is there anything else that needs changing? Pulled this ref and made the change, otherwise I could push it.

Reply:
except for the other comments i left here, no changes